.env.example

```diff
@@ -0,0 +1,14 @@
+# Django secret key used on app settings
+SECRET_KEY="your_secret_here"
+
+# authentication for koiki API.
+KOIKI_HOST="https://rekistest.koiki.es/services"
+KOIKI_AUTH_TOKEN="your_auth_token here"
+
+# wordpress woocommerce auth
```

Comment:
I'd rather more specific. Something like `Details to connect to WCFM, the Marketplace plugin's API on Woocommerce`.

Reply:
Fixed by: https://github.com/coopdevs/lazona_connector/pull/92/commits/2881bc3cc316a5113ff727b6609fb44e3b41c9e8